### ISSUE TITLE
FTDNN failed to convergence

### ISSUE THREAD

**Issue by chenllliang** [cvqluu/Factorized-TDNN#1]
Hello. I use the FTDNN model in your model.py to train the VoxCeleb1 dataset but the model works terribly compared to the original TDNN system, it can only reach 40% acc after 100 epochs however TDNN can reach nearly 100%.  And the loss is still relatively high after 100 epochs. It clearly failed to reach convergence. Have you tried this on Voxceleb1 dataset before? I hope to know the configuration. Thank you .

**Comment by cvqluu** [comment]
Yes the FTDNN model appears to work with me (when I use VoxCeleb 2 as training data). Are you making sure to take the orthogonal constraint step at each normal SGD step?

**Comment by cvqluu** [comment]
Closed until further info can be given
